data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1704 @@
+			branch resistance
+				has "Resistance: Questions: done"
+			choice
+				`	"What were all of them so worked up about?"`
+				`	"What's really going on with you and your company?"`
+			`	He looks into your eyes for a few seconds, before taking a long look around the hangars. "Talk inside your ship, we should." You two head inside, and you close the hatch. He sits down, and begins, "A transport company, I do not own. No company at all, I own. Transporting students, ranchers, immigrants, you were not." He looks at you intensely. "Saving lives, what you have been doing is. Saving them," he points to outside your ship.`
+			choice
+				`	"Saving them? From what?"`
+					goto what
+				`	"I just flew them from one planet to another."`
+			`	"And in doing so, saved their lives, you have," he responds. "The truth, it was, that arrive to the planets I sent you in time, my other ships would not, but not in a hurry due to contracts, we were. Searching for them, the Heliarchs were, and on their way, Punisher fleets were. Captured, all those you rescued would have been, if arrived before you, those Punishers had."`
+				goto reveal
+			label what
+			`	"Why, the Heliarchs. Our so-called 'protectors.' Paranoid, self-decorated butchers in their rooms deep within the rings..." He looks to the ground, disgust written on his face. "Preying on us for many centuries, they have been. With what insanities they invent and worship, their actions they justify."`
+			label reveal
+			choice
+				`	"The Heliarchs were looking for them? So they- you all are criminals?"`
+				`	"Who are you?"`
+			`	"Elirom, I am. Born on Saros. Raised there, fascinated by the Heliarchs since my infancy. When of age I came, into the Academy of Cadets I enlisted. Served the Heliarchy for 67 years, I had, before finally the right thing, I did. For the last 18 of those years, an arbiter, I was." He brings his hands together, as if to warm them up. "The Lunarium Redoubt, I then joined. Belong to that group, those you rescued do. Or, at least, their parents do... did." He sighs. "Criminals, outlaws... whatever names the Heliarch call us, simply trying to save our Coalition from them, we are. Saving people from their mock hunts for terrorists, my job is. Believe me, you may not. Choose to see us as what the Heliarchs deem us, you can. In doing that, though, changing the things they do, the things that seen, I have, the things that..." he trails off. "But, dead to them for decades now, I have been."`
```

Comment:
That's what I'm getting at there yeah.
Originally I was going to have Elirom be showing up every single mission with a different look, like a "master of disguise" sort of thing, but ultimately felt it would be more work than worth.
As for how he's not recognized, he wasn't involved in public events, rather in medical research/drug testing, so wouldn't be one of the more known arbiters. That, some good plastic surgery, and general cover from Houses Bebliss and Idriss to me seemed enough for him to manage to operate without being found out to be alive after all.

Reply:
Makes sense, but maybe you should provide the opportunity to ask him about it, since it seems like something that players would be wondering about.

Reply:
I think I'll save questions like that for later down the storyline.